web/Areas/Directory/Views/Index.cshtml

```diff
@@ -0,0 +1,151 @@
+﻿@using Viper.Classes.SQLContext;
+@model Viper.Models.AAUD.AaudUser
+@{
+    RAPSContext? rapsContext = (RAPSContext?)Context.RequestServices.GetService(typeof(RAPSContext));
+    IUserHelper UserHelper = new UserHelper();
+}
+@{
+    ViewData["Title"] = "Directory";
+}
+<h2>Directory</h2>
+<q-form @@submit="return false;">
+    <q-input class="q-ml-xs q-mr-xs"
+             dense
+             clearable
+             clear-icon="close"
+             outlined
+             debounce="300"
+             placeholder="Search for users"
+             v-model="userSearch"
+             autofocus>
+    </q-input>
+</q-form>
+
+<q-table title="Search Results"
+         dense
+         row-key="mothraId"
+         :loading="users.loading"
+         :rows="users.rows"
+         :columns="users.columns"
+         :pagination="users.pagination"
+         no-data-label="Search by first or last name"
+         no-results-label="No results found">
+    <template v-slot:body-cell-photo="props">
+        <q-td :props="props">
+            <q-img :props="props" :src="'https://viper.vetmed.ucdavis.edu/public/utilities/getbase64image.cfm?mothraID=' + props.row.mothraId + '&altphoto=1'" style="width:87px; height:111px" :id="'photo_' + props.row.mothraId"></q-img>
```

Comment:
There's a static function GetOldViperRootURL() in HttpHelper that could be used to get the url for Viper on the current server, instead of hard coding viper.vetmed.